Assets/_Laz/Scripts/Laz/LazCoordinatorBehaviour.cs

```diff
@@ -83,5 +83,9 @@ public void KillLaz()
         {
             _lazPlayer.KillLaz();
         }
+
+#if DEVELOPMENT_BUILD || UNITY_EDITOR
+        public LazMovementPropertyScriptableObject LazMovementPropertyScriptableObject => _movementPropertyScriptableObject;
```

Comment:
I'm curious. Instead of tacking on this thing into the LazCoordinator, 
can you just add `LazMovementPropertyScriptableObject` to the new WorldManager and grab it from that instead? would editing that also affect Laz's movement?